functionnal_tests/cypress/integration/publication/publications.spec.js

```diff
@@ -52,4 +52,24 @@ describe('Publications', () => {
     cy.get('.linkPreview__content__description').contains(fakePreview.description)
     cy.get(`.linkPreview__img[src="${fakePreview.image}"]`).should('be.visible')
   })
+
+  it('A translation button should be visible', function () {
+    const text = 'Hello, world'
+    cy.get('#wysiwygTimelineCommentPublication').type(text)
+    cy.get('button').contains('Publish').click()
+    cy.get('[data-cy=commentTranslateButton]').click()
+    cy.contains('.feedItem__publication', 'en')
+    cy.get('[data-cy=commentTranslateButton]').click()
+    cy.contains('.feedItem__publication', text)
+  })
+
+  it('a menu should allow to change the target language', () => {
+    const text = 'Hello, world'
+    cy.get('#wysiwygTimelineCommentPublication').type(text)
+    cy.get('button').contains('Publish').click()
+    cy.get('[data-cy=commentTranslateButton__languageMenu]').click()
+    cy.get('[data-cy=commentTranslateButton__language__fr]').click()
+    cy.get('[data-cy=commentTranslateButton]').click()
+    cy.contains('.feedItem__publication', 'fr')
+  })
 })
```

Comment:
I have two questions about this piece of code:
1. Could the constant, since it is the same, be declared in a higher scope and used in both tests?
2. Is there any particular reason why in one test you used arrow function and not in the other?
And a third question not linked to this code, a little more general:
3. It's not necessary to update the translations in this PR?

Reply:
1. yes, I'll do that
2. no special reason, I can stick to arrow functions.
3. you're right…

Reply:
Points addressed, there is a Portuguese translation missing now :innocent: .